### PR TITLE
Watch 04- Measure Heart Rate UI

### PR DESCRIPTION
**Pull request by sanketshah92**
- Design heart rate measure screen
- Adding animations for start listening
- Adding graph path to navigate from main menu to measure heart rate.

### CODE REVIEW

File: app/src/main/java/com/sanket/watchapplication/presentation/measureHR/MeasureHeartRateFragment.kt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupInitialView()
+    }
+
+    private fun setupInitialView() {
+        tv_measurement.visibility = View.VISIBLE
+        btnStart.visibility = View.VISIBLE
```
Comment (EntyR) [inline app/src/main/java/com/sanket/watchapplication/presentation/measureHR/MeasureHeartRateFragment.kt:30]: Please use view binding

File: app/src/main/res/drawable/rounded_bg_button.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+    <shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+            <solid android:color="#DB00FF" />
+            <corners android:radius="24dp" />
```
Comment (EntyR) [inline app/src/main/res/drawable/rounded_bg_button.xml:5]: Please extract resources

File: app/src/main/res/layout/fragment_measure_hr.xml [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        android:text="@string/start"
+        android:textColor="#000000"
+        android:textSize="12sp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/btnStop"
+        android:layout_width="72dp"
```
Comment (EntyR) [inline app/src/main/res/layout/fragment_measure_hr.xml]: Please extract all resources

**Review by EntyR — CHANGES_REQUESTED** [review]

**Comment by EntyR** [comment]
image

The background is from figma. You doesn't have to make it pixel by pixel but please make them similar. Also ask me if you need any background for ui debuging since you don't have access to figma, just put it as background for root element.

Animation is very nice! Still you will need to display real value there. It will be great if you could animate real value with custom view or something but you don't have what much time so can make it as simple chart and use some chart library like [this ](https://github.com/PhilJay/MPAndroidChart).

**Comment by EntyR** [comment]
Also I forgot to mention font used in app. I sent it as email. Please change it here and in other request

**Comment by sanketshah92** [comment]
> Also I forgot to mention font used in app. I sent it as email. Please change it here and in other request

done

**Comment by sanketshah92** [comment]
> image
> 
> The background is from figma. You doesn't have to make it pixel by pixel but please make them similar. Also ask me if you need any background for ui debuging since you don't have access to figma, just put it as background for root element.
> 
> Animation is very nice! Still you will need to display real value there. It will be great if you could animate real value with custom view or something but you don't have what much time so can make it as simple chart and use some chart library like [this ](https://github.com/PhilJay/MPAndroidChart).

fixed

File: app/src/main/res/layout/fragment_measure_hr.xml (hunk 3 above)
Comment (sanketshah92) [reply app/src/main/res/layout/fragment_measure_hr.xml]: done

File: app/src/main/res/drawable/rounded_bg_button.xml (hunk 2 above)
Comment (sanketshah92) [reply app/src/main/res/drawable/rounded_bg_button.xml:5]: done

**Pull request merged**